common/controllers/form-wizard.js

```diff
@@ -1,10 +1,51 @@
 const Sentry = require('@sentry/node')
 const { Controller } = require('hmpo-form-wizard')
-const { map, fromPairs } = require('lodash')
+const { flatten, map, fromPairs, forEach } = require('lodash')
 
 const fieldHelpers = require('../helpers/field')
 
 class FormController extends Controller {
+  middlewareSetup() {
+    super.middlewareSetup()
+    this.use(this.setupConditionalFields)
+  }
+
+  setupConditionalFields(req, res, next) {
+    const allFields = req.form.options.allFields
+    const stepFields = req.form.options.fields
+    const dependentFields = {}
+
+    forEach(stepFields, (field, key) => {
+      if (field.items) {
```

Comment:
I think you could have less nesting and simpler reasoning if you were to use a guard.

```js
if (!field.items) {
  return
}
// rest of the code
```

Reply:
I did consider that but the empty return and inconsistent behaviour sort of stopped me. 

I'll use the early return :)